sql/session.proto

```diff
@@ -42,4 +42,10 @@ message Session {
   // Indicates that the above transaction is mutating keys in the
   // SystemDB span.
   optional bool mutates_system_db = 4 [(gogoproto.nullable) = false, (gogoproto.customname) = "MutatesSystemDB"];
+  oneof timezone {
+    // A time zone, local, or default (same as local).
```

Comment:
Or it calls for defining local/default as UTC. We almost certainly want to allow `SET TIME ZONE DEFAULT` to reset to UTC if UTC is the default time zone the session starts in.